ipatests/test_integration/test_uninstallation.py

```diff
@@ -119,3 +119,8 @@ def test_failed_uninstall(self):
             self.master.run_command([
                 paths.DSCTL, serverid, 'remove', '--do-it'
             ])
+
+    @classmethod
+    def uninstall(cls, mh):
```

Comment:
Replica and master uninstallation are parts of testsuite, so we expect that once they can fail. In this case proper uninstallation will be needed.

Reply:
The issue is the following: if a test uninstalls the master, the base `uninstall` method is likely to fail as it tries to run ipa server-del on the master node. That's why I came up with this solution.
If you have a solution in mind, I would be happy to implement it.

Reply:
Maybe add raiseonerr=False to invocation of  server-del or specify two valid codes in ok_returncode?